### ISSUE TITLE
UndefVarError: Base.warn_once in deprecations.jl

### ISSUE THREAD

**Issue by jaketothepast** [JuliaText/TextAnalysis.jl#109]
This appears to be an issue with 1.0 compatibility, but I attempted to run the following
```julia
julia> remove_punctuation!(s[1])
ERROR: UndefVarError: warn_once not defined
Stacktrace:
 [1] remove_punctuation!(::StringDocument{String}) at /home/jake/.julia/packages/TextAnalysis/h7oB5/src/deprecations.jl:31
 [2] top-level scope at none:0
```

A cursory search of the Julia 1.0 docs did not prove fruitful looking for a Base.warn_once method. I searched github for Base.warn_once fixes and saw usage of the `Compat.@warn` macro, maybe we could use that?

I'm brand new to Julia and wouldn't mind tackling this issue if possible!

**Comment by cchderrick** [comment]
I think the proper way in 1.0 is ...
```julia
@warn "Test" maxlog=1
```
the `maxlog` control how many times it pops up, so in this case, it would be 1 for once.

**Comment by aviks** [comment]
Hi Jake, yes this is the codebase showing its age. It should be easy to fix, we'd welcome a PR.

**Comment by jaketothepast** [comment]
Wonderful, I'll work on it this weekend. Thanks for the response!

**Comment by aviks** [comment]
Thanks Jake, but Nick just made a this change in this #112

**Comment by aviks** [comment]
Closed by #112
